tests/units/hindawi/test_hindawi_parser.py

```diff
@@ -0,0 +1,167 @@
+        },
+        {
+            "raw_name": "Ortega, Pablo G.",
+            "affiliations": [
+                {
+                    "value": "Grupo de Física Nuclear and Instituto Universitario de Física Fundamental y Matemáticas (IUFFyM), Universidad de Salamanca, E-37008 Salamanca, Spain",
+                    "organization": "Grupo de Física Nuclear and Instituto Universitario de Física Fundamental y Matemáticas (IUFFyM), Universidad de Salamanca, E-37008 Salamanca",
+                    "country": "Spain",
+                }
+            ],
+        },
+        {
+            "raw_name": "Fernández, Francisco",
+            "affiliations": [
+                {
+                    "value": "Grupo de Física Nuclear and Instituto Universitario de Física Fundamental y Matemáticas (IUFFyM), Universidad de Salamanca, E-37008 Salamanca, Spain",
+                    "organization": "Grupo de Física Nuclear and Instituto Universitario de Física Fundamental y Matemáticas (IUFFyM), Universidad de Salamanca, E-37008 Salamanca",
+                    "country": "Spain",
+                }
+            ],
+        },
+    ]
+    for article in parsed_articles:
+        assert "authors" in article
+        assert authors == article["authors"]
+
+
+def test_abstracts(parsed_articles):
+    for article in parsed_articles:
+        assert "abstract" in article
+        assert (
+            article["abstract"]
+            == "In recent years, the discovery in quarkonium spectrum of several states not predicted by the naive quark model has awakened a lot of interest. A possible description of such states requires the enlargement of the quark model by introducing quark-antiquark pair creation or continuum coupling effects. The unquenching of the quark models is a way to take these new components into account. In the spirit of the Cornell Model, this is usually done by coupling perturbatively a quark-antiquark state with definite quantum numbers to the meson-meson channel with the closest threshold. In this work we present a method to coupled quark-antiquark states with meson-meson channels, taking into account effectively the nonperturbative coupling to all quark-antiquark states with the same quantum numbers. The method will be applied to the study of the X(3872) resonance and a comparison with the perturbative calculation will be performed."
+        )
+
+
+def test_title(parsed_articles):
+    for article in parsed_articles:
+        assert "title" in article
+        assert (
+            article["title"]
+            == "Unquenching the <italic>Quark <b>Model</b></italic> in a Nonperturbative Scheme"
+        )
+
+
+def test_date_published(parsed_articles):
+    for article in parsed_articles:
+        assert "date_published" in article
+        assert article["date_published"] == "2019-05-02"
+
+
+def test_page_nr(parsed_articles):
+    for article in parsed_articles:
+        assert "page_nr" in article
+        assert article["page_nr"] == [7]
```

Comment:
no need to assert both :) if `article['page_nr']` doesn't exist it will fail

Reply:
but if it exists, but the value is different?